### PR TITLE
Backend comment

### PR DESCRIPTION
**Pull request by AlexisFLT**

### CODE REVIEW

**Review by Sax593 — CHANGES_REQUESTED** [review]
Please add only the files that concern the comments

**Review by Sax593 — DISMISSED** [review]

File: backend/src/router.js [hunk 1]
```diff
@@ -2,12 +2,28 @@ const express = require("express");
...
+// router.put("/suggests/:id", suggestsControllers.edit);
+// router.delete("/suggests/:id", suggestsControllers.destroy);
+
+// const usersControllers = require("./controllers/usersControllers");
+
+// router.get("/users", usersControllers.browse);
+// router.get("/users/:id", usersControllers.read);
+// router.post("/users", usersControllers.add);
+// router.put("/users/:id", usersControllers.edit);
+// router.delete("/users/:id", usersControllers.destroy);
```
Comment (LoicBrassart) [inline backend/src/router.js]: Dead code... You just didn't read this, did you ? -_-

**Review by LoicBrassart — CHANGES_REQUESTED** [review]
Read your PRs carefully plz

**Review by Sax593 — APPROVED** [review]

**Review by LoicBrassart — APPROVED** [review]
SnE

**Pull request merged**